### PR TITLE
add com.github.flxzt.rnote

### PR DESCRIPTION
**Pull request by flxzt**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. My app won't run without session-bus permissions
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub.
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by flxzt** [comment]
bot, build com.github.flxzt.rnote

File: com.github.flxzt.rnote.json [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    "runtime-version": "41",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "rnote",
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--socket=session-bus",
```
Comment (hfiguiere) [inline com.github.flxzt.rnote.json:13]: why do you need session bus access? You should just declare which interface you need to access.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by barthalion** [comment]
bot, build com.github.flxzt.rnote

**Comment by barthalion** [comment]
It launches fine, but I can't find a way to actually draw anything. Is there some catch or is it a bug?

**Comment by flxzt** [comment]
Have you enabled "mouse drawing" from the secondary menu?

**Comment by barthalion** [comment]
Ah, that makes sense, I missed that option somehow. Can confirm it works, thanks.

/merge

**Pull request closed without merging**